### PR TITLE
Changed favicon to png with transparent background

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* None

*Description of changes:*
- app version = 2.4.16
- changed favicon.ico -> favicon.png

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/index.html [hunk 1]
```diff
@@ -6,8 +6,7 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <title>BC Business Registry</title>
     <meta name="Description" content="File your BC cooperative association's annual reports and maintain your registered office addresses and director information.">
-    <link rel="icon" type="image/png" href="static/images/favicon.ico"/>
-    <link rel="icon" href="favicon.ico">
```
Comment (severinbeauvais) [inline auth-web/index.html:10]: Line 9 wasn't needed.
Line 10 now refers to PNG.

File: package-lock.json
Comment (severinbeauvais) [inline package-lock.json:1]: This file, in sbc-auth/, doesn't belong. The correct file (as committed above) is `sbc-auth/auth-web/package-lock.json`.

File: auth-web/public/favicon.png
Comment (severinbeauvais) [inline auth-web/public/favicon.png:1]: The is the same image file used by Filings UI, Namerequest UI, bcregistry, etc. It has a transparent background.

**Review by seeker25 — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2553-eln1005a.web.app

**Comment by severinbeauvais** [comment]
A couple of things to notice here...

In Chrome, new version (temp url) in first tab displays bcgov logo with transparent background. Old version (dev) sometimes does not display logo (3rd tab), or displays logo with white background (4th tab):

image

In Firefox, new version (temp) in first tab displays properly. Old version (dev) in last tab doesn't show logo:

image

**Comment by severinbeauvais** [comment]
The favicon in dev now has the transparent background, but it _still_ doesn't display every time !?!?

1a. go to MBR
1b. observe that icon is correct
2. manage a business
3a. click on breadcrumb to return to MBR
3b. observe that icon is missing!

**Comment by severinbeauvais** [comment]
> The favicon in dev now has the transparent background, but it _still_ doesn't display every time !?!?
> 
> 1a. go to MBR 1b. observe that icon is correct 2. manage a business 3a. click on breadcrumb to return to MBR 3b. observe that icon is missing!

Ohhhh... It's fine if you go to the decide-business (top) page first, but if you go to the MBR first, the icon is missing.

**Pull request merged**